app/controllers/doorkeeper_applications_controller.rb

```diff
@@ -21,7 +21,18 @@ def update
 
   def users_with_access
     query = policy_scope(User).with_access_to_application(@application)
-    @users = query.page(params[:page]).per(100)
+    @users = query.page(params[:page]).per(10)
+  end
+
+  def access_logs
+    smokey_uids = User.where("name LIKE 'Smokey%'").pluck(:uid)
+    @logs = @application.event_logs
+      .includes(:user)
+      .where(event_id: 47)
```

Comment:
Is `47` the successful user application event? Do you think it's worth storing this under a descriptive variable name or some kind of enum to make this as obvious as possible?

Reply:
Ah yeah, it's there as `EventLog::SUCCESSFUL_USER_APPLICATION_AUTHORIZATION.id` I just couldn't get it to work originally because my ruby sux. (It's [defined here](https://github.com/alphagov/signon/blob/af0bd6f3bb545ba7c29ca363c8f112d92c1f9570/app/models/event_log.rb#L6-L53), and I couldn't work out the syntax - thought it would be `EventLog::EVENTS::SUCCESSFUL_USER_APPLICATION_AUTHORIZATION` but of course it isn't).